shell/platform/android/android_shell_holder.cc

```diff
@@ -135,7 +139,27 @@ AndroidShellHolder::AndroidShellHolder(
         FML_LOG(ERROR) << "Failed to set Workers task runner priority";
       }
     });
+
+#if defined(__ANDROID_API__) && __ANDROID_API__ >= 28
+    if (IsNDKImageDecoderAvailable()) {
+      shell_->RegisterImageDecoder(
+          [](sk_sp<SkData> buffer) {
+            auto generator = SkImageGeneratorNDK::MakeFromEncodeNDK(buffer);
+            return BuiltinSkiaImageGenerator::MakeFromGenerator(
+                std::move(generator));
+          },
+          -1);
+      FLM_LOG(INFO) << "Registered NDK image decoder (API level 30+)";
```

Comment:
Remove log. DLOG is fine for this if you want though.

Reply:
Removed this branch.